Week01/Problem04/qxu_04.py

```diff
@@ -0,0 +1,16 @@
+palindromic = 0
+for ii in range(100,1000):
+	for jj in range(ii,1000):
+		prod = ii * jj
+		prodstr = str(prod)
+		flag = 1
+		for d in range(len(prodstr)//2):
```

Comment:
You could also check that the `prodstr` is a palindrome by the following:
```
prodstr == prodstr[::-1]
```
Since `prodstr` is a string, and a string is simply a list of characters, you can use list slicing to reverse the order of the string.